src/main/java/com/minecolonies/coremod/util/BlockUtils.java

```diff
@@ -186,7 +186,7 @@ public static boolean isWater(final IBlockState iBlockState)
     public static boolean isBlockSeed(@NotNull final World world, @NotNull final BlockPos pos)
     {
         final ItemStack stack = BlockUtils.getItemStackFromBlockState(world.getBlockState(pos.up()));
-        return  stack != null && stack.getItem() instanceof ItemSeeds;
+        return !ItemStackUtils.isEmpty(stack) && stack.getItem() instanceof ItemSeeds;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "stack" is nullable here [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2259)